src/app/base/components/node/FieldlessForm/FieldlessForm.tsx

```diff
@@ -58,9 +66,11 @@ export const FieldlessForm = <E,>({
         const [, actionFunction] =
           Object.entries(actions).find(([key]) => key === actionMethod) || [];
 
+        const params =
+          action === NodeActions.SOFT_OFF ? { stop_mode: "soft" } : {};
```

Comment:
It would be cleaner if this was moved to soft power off action that you created in `src/app/store/machine/slice.ts`